pkg/proxy/ipvs/proxier.go

```diff
@@ -614,10 +614,18 @@ func (em proxyEndpointsMap) unmerge(other proxyEndpointsMap) {
 }
 
 // CanUseIPVSProxier returns true if we can use the ipvs Proxier.
-// This is determined by checking if all the required kernel modules are loaded. It may
+// This is determined by checking if all the required kernel modules can be loaded. It may
 // return an error if it fails to get the kernel modules information without error, in which
 // case it will also return false.
 func CanUseIPVSProxier() (bool, error) {
+	// Try to load IPVS required kernel modules using modprobe
+	for _, kmod := range ipvsModules {
+		err := utilexec.New().Command("modprobe", "--", kmod).Run()
+		if err != nil {
+			glog.Warningf("Failed to load kernel module: %v", kmod)
```

Comment:
nit:

The log message is identical with https://github.com/vfreex/kubernetes/blob/5a0588c34d13c77166d584c61aa59d1621ddb191/pkg/proxy/ipvs/proxier.go#L642

I would prefer a verbose message here, e.g. "Failed to load kernel module: %v using modprobe, please ignore it when kube-proxy is running inside container without mounting /lib/modules". WDYT ^-^

BTW, please fix the log message https://github.com/vfreex/kubernetes/blob/5a0588c34d13c77166d584c61aa59d1621ddb191/pkg/proxy/ipvs/proxier.go#L642, what about "Required kernel module %s is not loaded"?